enterprise-suite/scripts/lbc.py

```diff
@@ -214,11 +215,40 @@ def check_credentials(creds):
         else:
             printerr('error: check_credentials failed: {}'.format(err))
     finally:
-        return success 
+        return success
 
-def preinstall_check(creds, minikube=False, minishift=False):
+
+
+url_http_timeout=3 # 3 seconds
+installer_url="https://raw.githubusercontent.com/lightbend/console-charts/master/enterprise-suite/scripts/lbc.py"
+warning_display_timeout=2 # 2 seconds
```

Comment:
Global constants like these should be in uppercase, just to be consistent with existing code. However, since they are only used in check_new_install_script() and not something that is expected to be changed, maybe they don't have to be global and can be limited to the function scope.